docs/roman/dq_init/description.rst

```diff
@@ -0,0 +1,28 @@
+Description
+============
+The Data Quality (DQ) initialization step in the calibration pipeline
+populates the DQ mask for the input dataset. Flag values from the
+appropriate static mask ("MASK") reference file in CRDS are copied into the
+"PIXELDQ" array of the input dataset, because it is assumed that flags in the
+mask reference file pertain to problem conditions that affect all groups and
```

Comment:
Not that anyone would get confused, 'all ... integrations' is for the single integration of course.

Reply:
Removed the integrations.